core/tests/protractor_utils/users.js

```diff
@@ -53,7 +53,13 @@ var logout = function() {
 // The user needs to log in immediately before this method is called. Note
 // that this will fail if the user already has a username.
 var _completeSignup = function(username) {
+  // This is required since there is a redirect which can be considered
+  // as a client side navigation and the tests fail since Angular is
+  // not found due to the navigation interfering with protractor's
+  // bootstrapping.
+  browser.waitForAngularEnabled(false);
```

Comment:
@ankita240796 is it required even if the redirected page is `angular`?

Reply:
No, if the redirected page is angular, we should not have this line.